projects/components/src/titled-content/titled-content.component.ts

```diff
@@ -10,7 +10,7 @@ import { TitledHeaderControlDirective } from './header-controls/titled-header-co
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <div class="titled-content-container">
-      <div class="header">
+      <div class="header" [htLayoutChangeTrigger]="this.shouldShowHeader">
```

Comment:
Using this trigger here fails a lot of test. We now have to provide layout change service almost everywhere. Mocking the directive didn't help. Was wondering if we could avoid it.

Reply:
It didn't. It somehow still asking for LayoutService. Need to look more

Reply:
Added a new mockDashboardWidgetProviders.